src/components/MainCardItem/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { Link } from "react-router-dom"
+import { IFilms } from "../../types/IFilms"
+import { prepareGenre } from "../../utils/prepareGenre"
+
+import './styles.scss'
+
+interface PropsParams {
+    film: IFilms
+}
+
+const MainCardItem: React.FC<PropsParams> = ({film}) => {
```

Comment:
не используй FC тип, используй такую краткую запись определения пропсов
```suggestion
const MainCardItem = ({film}: PropsParams) => {
```